backend/build.gradle

```diff
@@ -13,9 +13,18 @@ repositories {
 }
 
 dependencies {
+	// core infrastructure
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+
+	// data layer dependencies
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation "com.vladmihalcea:hibernate-types-52:2.5.0"
+	implementation "org.liquibase:liquibase-core"
```

Comment:
Are these versions specified in the dependency management pom?

Reply:
Yep! Or I should say, by way of the dependency management plugin, through a process that I do not need to know the internals of, so I don't. 😬